stdlib/Makefile

```diff
@@ -55,16 +55,28 @@ endif
 allopt: stdlib.cmxa std_exit.cmx
 opt.opt: allopt
 
+.PHONY: check-stale
+check-stale:
+# On cygwin globasciiranges is unset so this glob also match A-Z. We disable
+# it only on this shell command.
+	2>/dev/null shopt -s globasciiranges || true; \
+  for files in '$(INSTALL_LIBDIR)'/stdlib__[a-z]*.cm*; do \
+    case $$files in \
+    '$(INSTALL_LIBDIR)/stdlib__[a-z]*.cm*') \
+    ;; \
+    *) \
+      echo "'"'$(INSTALL_LIBDIR)'"'"' contains stale stdlib artefacts \
+please execute:'; \
+      echo '    rm '"'"'$(INSTALL_LIBDIR)'"'"'/stdlib__[a-z]*.cm*'; \
+      echo 'and re-run make install'; \
```

Comment:
This command might delete more file that necessary on Cygwin because we don't add `shopt` command.
Also instead of `re-run make install` I could have printed the original command If makefile weren't recursive: https://stackoverflow.com/a/38754878/7227940
One more point for "Recursive Make Considered Harmful". Do you want I help to de-recursify Makefile ?